js-api/src/chem.ts

```diff
@@ -359,6 +376,15 @@ export namespace chem {
       return clearButton;
     }
 
+    updateInvalidMoleculeWarning(error?: string) {
+      ui.empty(this.invalidMoleculeWarning);
+      if(error) {
+        const errorDiv = ui.divText('Malformed molecule');
+        ui.tooltip.bind(errorDiv, error);
```

Comment:
This would create event subscription every time you call it. Bind it once to a function (hope JS API allows it)

Reply:
done